command/server/server_command.go

```diff
@@ -143,14 +143,22 @@ func (c *ServerCommand) DefineFlags() {
 		FlagOptional: true,
 	}
 
-	c.FlagMap["max-peers"] = helper.FlagDescriptor{
-		Description: fmt.Sprintf("Sets the client's max peer count. Default: %d", helper.DefaultConfig().Network.MaxPeers),
+	c.FlagMap["max-inbound-peers"] = helper.FlagDescriptor{
```

Comment:
Where is the max-peers flag?

You've removed it, and only left max-outbound-peers and max-inbound-peers